corehq/messaging/scheduling/scheduling_partitioned/models.py

```diff
@@ -679,3 +679,13 @@ class Meta(AbstractTimedScheduleInstance.Meta):
         unique_together = (
             ('case_id', 'timed_schedule_id', 'recipient_type', 'recipient_id'),
         )
+
+    def additional_deactivation_condition_reached(self):
+        return_condition = super().additional_deactivation_condition_reached()
```

Comment:
nit: can there be a better name for this?
`additional_deactivation_condition_reached = super().additional_deactivation_condition_reached()` may be?

Reply:
fixed. [279328f](https://github.com/dimagi/commcare-hq/pull/32671/commits/279328fd2c8f115893f6817e1ba0ac7731592cde)